docs/specs/helpers.md

```diff
@@ -44,6 +42,100 @@ The anchor point is highlighted with an orange dot.
     </script>
 </lottie-playground>
 
+Transforms the parent's coordinate system.
+
+When calculating a transform matrix, properties SHOULD be applied as follows:
+
+1. Translate by $-a$
+1. Scale by $\frac{s}{100}$
+1. If $sk \neq 0$:
+    1. Rotate by $-sa$
+    1. Skew x by $\tan(-sk)$
+    1. Rotate by $sa$
+1. Rotate by $-r$
+1. Translate by $p$
+
+Assuming a transform matrix with the following layout, with the labels equivalent to the
+[CSS matrix transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix):
+
+$$
+\begin{pmatrix}
+a & b & 0 & 0 \\
+c & d & 0 & 0 \\
+0 & 0 & 1 & 0 \\
```

Comment:
We should probably spec as a 3x3 matrix instead. 4x4 is more future-proof (3D layers), but at this point the third row and column are always going to be [0,0,1,0].  Let's worry about 3D transforms when we get there.

(CSS uses 4x4 because it supports 3D transforms, SVG uses 3x3)